group_project_v2/utils.py

```diff
@@ -281,3 +283,7 @@ def add_resource(block, resource_type, path, fragment, via_url=False):
         action_parameter = loader.load_unicode(path)
 
     action(action_parameter)
+
+
+def get_block_content_id(block):
```

Comment:
This one-line function is used only twice and seems too trivial to be factored out into "utils".